client/components/external-link/index.jsx

```diff
@@ -41,6 +42,10 @@ export default React.createClass( {
 			rel: 'external'
 		} );
 
+		if ( props.target ) {
+			props.rel = props.rel.concat( ' noopener noreferrer' );
```

Comment:
TIL [`String#concat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat). Then again, this could have just been `props.rel += ' noopener noreferrer';`

No need to change anything.